tests/lib/rules/no-restricted-properties.js

```diff
@@ -75,6 +75,21 @@ ruleTester.run("no-restricted-properties", rule, {
                 object: "obj",
                 property: "foo"
             }]
+        }, {
+            code: "foo.bar",
+            options: [{
+                property: "baz"
+            }]
+        }, {
+            code: "foo.bar",
+            options: [{
+                object: "baz"
+            }]
+        }, {
+            code: "foo()",
```

Comment:
@ljharb I agree, but the function call confused me, thought maybe there was special handling for `foo`.

@not-an-aardvark Please! :smile: